src/performance/Processor.cpp

```diff
@@ -651,6 +646,10 @@ void Processor::pollAndProcessNewScores()
 		if (scoreIt == std::end(user.Scores()))
 		{
 			tlog::warning() << StrFormat("Could not find score ID {0} in result set.", scoreId);
+
+			// even though the score wasn't processed, we still want to mark the queue as completed.
+			_pDB.NonQuery(StrFormat("UPDATE score_process_queue SET status = 1 WHERE mode = {0} AND score_id = {1};", static_cast<int>(_gamemode), _scoreId));
```

Comment:
Again, likely not important, but the ";" at the end of your query is redundant with the way this program executes SQL statements.